Require use of https when using local dev override (#4865)
camptocamp/c2cgeoportal: doc/developer/debugging.rst

Open the application at the following URL: ``https://<host>/<entry_point>/dev/<interface>.html``.

Pyramid
-------

If the ``pyramid_debugtoolbar`` is enabled, the error is directly shown in the query that fails.

Authentication
--------------

For security issue the cookie is accessible only in http (meens not JavaScript),

> and is secure (means send on https request, not http).

Reviewer: ```suggestion
and is secure (means sent on https request, not http).
```

> For that and some other issue we should have our application on https also for the devlopement.

Reviewer: ```suggestion
For that and for some other issues we should have our application on https also for development.
```

> To do that you just should, if it's not already done copy the file ``docker-compose.override.sample.yaml``

Reviewer: ```suggestion
To do that you should, if it's not already done, just copy the file ``docker-compose.override.sample.yaml``
```

to ``docker-compose.override.yaml``.

> Then access to the application on `https://172.17.0.1:8484/ <https://172.17.0.1:8484/>`_.

Reviewer: ```suggestion
Then access the application on `https://172.17.0.1:8484/ <https://172.17.0.1:8484/>`_.
```
